builtin/settingtypes.txt

```diff
@@ -469,6 +469,9 @@ enable_waving_plants (Waving plants) bool false
 
 [***Advanced]
 
+#    Arm inertia
```

Comment:
longer description would be nice, something like
`more realistic movement of arm when turning view`